### ISSUE TITLE
Upgraded to 8.05

### ISSUE THREAD

**Issue by b1063n** [uPesy/easyeda2kicad.py#147]
It seems the symbol library does not work anymore in 8.05

Can this change be related to this?

https://gitlab.com/kicad/code/kicad/-/issues/18250

**Comment by b1063n** [comment]
Nevermind, somehow in the xxx.kicad_sym file an extra ")" was added at some point somehow and it did not allow for further addition of parts.

Manually removed it and it seems fine now.

**Comment by b1063n** [comment]
Yes that is how I fixed it. It happens occasionally (not very often), the fix is rather easy so I dont mind it too much.

**Comment by uPesy** [comment]
Fix in #184
